jaeger-production-template.yml

```diff
@@ -48,6 +48,12 @@ items:
           volumeMounts:
           - name: jaeger-configuration-volume
             mountPath: /conf
+          env:
+          - name: SPAN_STORAGE
+            valueFrom:
+              configMapKeyRef:
+                name: jaeger-configuration
```

Comment:
How does it work it will set the value based on `configMap` key?

Reply:
Yes, it gets the value from the configmap, but sets it as an environment variable.

Reply:
nice feature :+1: